Restore build version information

starksm64 opened the pull request:
- Update the com.sun.tdk.signaturetest.Version class to get version info from the artifact jar manifest.
- Remove the obsolete release profile used in jboss fork
- Fixes #33

dmatej reviewed (approved):
Note - build timestamps are in conflict with repeatable builds. We had a long discussion about that in GlassFish; if you don't change sources, the build should produce an artifact with the same checksum. But when time changed ... checksum changed too. GlassFish now adds just the commit id, time stamps were abandoned.

starksm64 commented:
> Note - build timestamps are in conflict with repeatable builds. We had a long discussion about that in GlassFish; if you don't change sources, the build should produce an artifact with the same checksum. But when time changed ... checksum changed too. GlassFish now adds just the commit id, time stamps were abandoned.

Yeah, I had already seen that in the VersionIT test. I switched this over to use the git info as suggested and simplified the code.

scottmarlow commented:
Nice work Scott!
